sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -380,7 +384,63 @@ for (int i = 0; i < result.Tables.Count; i++)
 
 For more information and samples see [here][analyze_prebuilt_document].
 
+### Use the Prebuilt Read Model
+Analyze textual elements, such as page words and lines, styles, and text language information from documents using the prebuilt read model.
 
+```C# Snippet:FormRecognizerAnalyzePrebuiltReadFromUriAsync
+string fileUri = "<fileUri>";
+
+AnalyzeDocumentOperation operation = await client.StartAnalyzeDocumentFromUriAsync("prebuilt-read", fileUri);
+
+await operation.WaitForCompletionAsync();
+
+AnalyzeResult result = operation.Value;
+
+Console.WriteLine("Detected languages:");
+
+foreach (DocumentLanguage language in result.Languages)
+{
+    Console.WriteLine($"  Found language '{language.LanguageCode}' with confidence {language.Confidence}.");
```

Comment:
not for this PR, but it might be nice to use the spans or a get_language helper to also output the word/line that was found in the language